packages/synapse-react-client/src/components/CreateTableViewWizard/CreateTableViewWizard.integration.test.tsx

```diff
@@ -0,0 +1,559 @@
+import React from 'react'
+import { act, render, screen, waitFor, within } from '@testing-library/react'
+import { server } from '../../mocks/msw/server'
+import CreateTableViewWizard, {
+  CreateTableViewWizardProps,
+} from './CreateTableViewWizard'
+import { createWrapper } from '../../testutils/TestingLibraryUtils'
+import mockProjectEntityData from '../../mocks/entity/mockProject'
+import userEvent from '@testing-library/user-event'
+import { getCreateColumnModelBatchHandler } from '../../mocks/msw/handlers/tableQueryHandlers'
+import {
+  ENTITY_VIEW_CONCRETE_TYPE_VALUE,
+  ENTITY_VIEW_TYPE_MASK_FILE,
+  ENTITY_VIEW_TYPE_MASK_FOLDER,
+  ENTITY_VIEW_TYPE_MASK_PROJECT,
+  MATERIALIZED_VIEW_CONCRETE_TYPE_VALUE,
+  SUBMISSION_VIEW_CONCRETE_TYPE_VALUE,
+  TABLE_ENTITY_CONCRETE_TYPE_VALUE,
+  VIRTUAL_TABLE_CONCRETE_TYPE_VALUE,
+} from '@sage-bionetworks/synapse-types'
+import { addColumnModelToForm } from '../TableColumnSchemaEditor/TableColumnSchemaEditorTestUtils'
+import SynapseClient from '../../synapse-client'
+import {
+  MOCK_ACCESS_TOKEN,
+  MOCK_CONTEXT_VALUE,
+} from '../../mocks/MockSynapseContext'
+import { EntityFinderModal } from '../EntityFinder/EntityFinderModal'
+import defaultFileViewColumnModels from '../../mocks/query/defaultFileViewColumnModels'
+
+jest.mock('../EntityFinder/EntityFinderModal', () => ({
+  EntityFinderModal: jest.fn(() => (
+    <div data-testid="EntityFinderModalMocked" />
+  )),
+}))
+
+const mockEntityFinderModal = jest.mocked(EntityFinderModal)
```

Comment:
Mocking just the entity finder since it's hard to interact with it in @testing-library/react